.github/actions/install-package-deps/action.yml

```diff
@@ -20,7 +26,26 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - id: download-rClr
+      if: ${{ inputs.install-rclr }}
+      run: |
+        RCLR_URL <- "https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v${{ inputs.rClr-version }}/rClr_${{ inputs.rClr-version }}.zip"
+        download.file(RCLR_URL, "rClr_${{ inputs.rClr-version }}.zip")
+      shell: Rscript {0}
+    - id: install-package-dependencies
+      if: ${{ inputs.install-rClr }}
```

Comment:
Why is this step executed only if `install-rClr`=true?
We can have the situations where rClr is not required but there are another R-dependencies (e.g. in the packages tlf or ospsuite.utils)

Reply:
ah, ok, in the next step you install everything except rClr if `install-rClr` is set to false.

Reply:
see reply below.